buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -618,7 +618,26 @@ def set_build_variables() {
 def set_sdk_variables() {
     DATESTAMP = get_date()
     SDK_FILE_EXT = SPEC.contains('zos') ? '.pax' : '.tar.gz'
-    SDK_FILENAME =  "OpenJ9-JDK${SDK_VERSION}-${SPEC}-${DATESTAMP}${SDK_FILE_EXT}"
+    // Get full version from tag (eg. 11.0.8+10) and replace + with _
+    FULL_SDK_VERSION = sh (
```

Comment:
`FULL_SDK_VERSION ` is global variable and already set in `set_sdk_versions()`. This is not required.

Reply:
This was a result of a bad merge resolution. Will update.